pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNativeV2.java

```diff
@@ -394,14 +395,14 @@ public Map<String, String> getTraceInfo() {
   }
 
   @Override
-  public void setReplicaGroups(@NotNull Set<Integer> replicaGroups) {
-    _replicaGroups = replicaGroups;
+  public void setPools(@NotNull Set<Integer> replicaGroups) {
```

Comment:
Please also check other places
```suggestion
  public void setPools(@NotNull Set<Integer> pools) {
```